docs/server.rst

```diff
@@ -184,3 +184,48 @@ After you configured Tunneldigger, you can run the broker::
 
     cd /srv/tunneldigger
     /srv/env_tunneldigger/bin/python -m tunneldigger_broker.main /srv/tunneldigger/broker/l2tp_broker.cfg
+
+Docker Image
+------------
+
+There is a docker image available from `wlanslovenija/tunneldigger-broker
+<https://hub.docker.com/r/wlanslovenija/tunneldigger-broker>`_.
+You can run it when you installed docker.
+Also, you need to install all the kernel modules mentioned at the start
+of the document because docker images share the kernel of the host system,
+including the modules ``ebtables`` and ``sch_netem``
+and activate these kernel modules.
+
+.. code:: shell
+
+    docker run --rm -it --privileged wlanslovenija/tunneldigger-broker
+
+You can configure the different values in the config file using
+environment variables as shown here:
+
+.. code:: shell
+
+    docker run -e address=127.0.0.1 -e port=53,123,8942 \
+               -e interface=lo      -e max_tunnels=1024 \
+               -e port_base=20000   -e tunnel_id_base=100 \
+               -e namespace=default -e connection_rate_limit=10 \
+               -e pmtu=0            -e verbosity=DEBUG \
+               -e log_ip_addresses=false --privileged \
+               -it wlanslovenija/tunneldigger-broker
+
+Instead of configuring the environment variables one by one
+using the ``-e`` option, you can also use the ``--env-file``
+option.
```

Comment:
Same question as for the client image: when is this server image *useful*? If the answer is "mostly for documentation and testing", that's okay, but then it should say so.